### DISCUSSION TITLE
Welcome to muslimboard Discussions!

### DISCUSSION THREAD

**Discussion by novalagung** [novalagung/muslimboard#71] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by safrony** [comment]
Assalamu'alaikum warahmatullahi wabarakatuh...

Thank you @novalagung for this extension that i hope someone will create..

i want to share my opinion about the design of todo list, i think it will be more elegant if we can make it more compact, or create option so we can choose our preference about the design..

**Reply by novalagung** [reply]
Wa'alaikumsalam warohmatullahi wabarokatuh

Hi mas @safrony , making it more compact sounds good to me. It should be possible to do that. However I'm not good with UI stuff so I'm not exactly sure what changes there that I should apply. Perhaps it would be nice to have some reference or design for it.

**Comment by Dekymulyana** [comment]
Assalamu'alaikum warahmatullahi wabarakatuh...
Terima kasih @novalagung atas tools yang sangat keren dan bermanfaat ini...
Apakah saya bisa minta quotes-quotes nya atau sumber api nya ? terima kasih
